src/test/java/seedu/address/logic/commands/room/InitRoomCommandTest.java

```diff
@@ -69,9 +70,10 @@ void execute_decreaseNumberOfRooms_success() {
     @Test
     void execute_increaseNumberOfOccupiedRooms_success() {
         RoomList roomList = TypicalRooms.getTypicalRoomList();
+        RoomList roomList1 = TypicalRooms.getTypicalRoomList();
+
         Model model = new ModelManager(new PatientRecords(), roomList, new UserPrefs());
-        Model expectedModel =
-                new ModelManager(model.getPatientRecords(), roomList, new UserPrefs());
+        Model expectedModel = new ModelManager(model.getPatientRecords(), roomList1, new UserPrefs());
```

Comment:
I think you can use rename `roomList1` to `expectedRoomList` to highlight the difference between the 2 `RoomList`. Same for the other test cases!

Reply:
Okay thanks